embassy-nrf/src/pwm.rs

```diff
@@ -216,6 +216,13 @@ impl<'d, T: Instance> SequencePwm<'d, T> {
         r.tasks_stop.write(|w| unsafe { w.bits(0x01) });
     }
 
+    /// Enables the PWM generator.
+    #[inline(always)]
+    pub fn enable(&self) {
```

Comment:
Hrm. The fn called `enable` is again coming from that embedded hal trait for the simplepwm

The enable fn isnt necessary for sequence because the constructor and start enable, and theres no way to disable. That said, I think ill add a disable to stop fn in my pr, maybe theres some power savings there, not sure. 

I guess technically we could rename start and stop to enable and disable but Im not that excited about that, since the do very different things and should probably have different names.

Reply:
> Hrm. The fn called `enable` is again coming from that embedded hal trait for the simplepwm
> 
> The enable fn isnt necessary for sequence because the constructor and start enable, and theres no way to disable. That said, I think ill add a disable to stop fn in my pr, maybe theres some power savings there, not sure.
> 
> I guess technically we could rename start and stop to enable and disable but Im not that excited about that, since the do very different things and should probably have different names.

You already had a `disable` function... are you stating that you don't need `disable` either?

Reply:
pwmsequence start and stop are disable and enable. You could argue they could be called enable and disable, but they do a good bit more than enable and disable so maybe different naming is good. but then yes its confusing it had a disable fn at all so yes I just got rid of it a few min ago